calico/felix/test/test_refcount.py

```diff
@@ -120,6 +193,10 @@ def on_object_cleanup_complete(self, *args, **kwargs):
         super(RefMgrForTesting, self).on_object_cleanup_complete(*args,
                                                                  **kwargs)
 
+    @actor_message()
```

Comment:
This doesn't need to be an actor_message.  I've tidied up the docs under #500 to reflect that.

(It's safe to make it one but it'll get called from the Actor's greenlet so it'll bypass the queue anyway.)